GoogleDataTransport/GDTCCTLibrary/GDTCOREvent+GDTCCTSupport.m

```diff
@@ -104,7 +109,8 @@ - (void)setNetworkConnectionInfoData:(NSData *)networkConnectionInfoData {
                                                      options:0
                                                        error:&error] mutableCopy];
         if (error) {
-          GDTCORLogDebug(@"Error when setting an even'ts event_code: %@", error);
+          GDTCORLogError(GDTCORMCESetEventInfoError,
```

Comment:
typo

Reply:
What Paul said is the case, but it's up to you how to move forward.

Reply:
Sounds good - we talked in person and decided to go in favor of this change: https://github.com/google/GoogleDataTransport/pull/54